api/spec/src/types.tsp

```diff
@@ -103,6 +114,31 @@ model Resource {
   ...ResourceTimestamps;
 }
 
+/**
+ * Represents a resource that doesn't exist in the persistence layer, just a representation.
+ */
+@friendlyName("VirtualResource")
+model VirtualResource {
+  ...OmitProperties<Resource, "id">;
+  ...Keyed;
+}
+
+/**
+ * Represents resources that can be cadenced, have scheduled activity changes.
+ */
+@friendlyName("CadencedResource")
+model CadencedResource {
```

Comment:
Would love to have a settlement on the naming used for describing time period, so I can also update the plan api.

Reply:
sure, I believe this is something we talked about around early September, but if you have any preferences lets change accordingly